src/js_native_api_v8.h

```diff
@@ -3,6 +3,7 @@
 
 // This file needs to be compatible with C compilers.
 #include <string.h>  // NOLINT(modernize-deprecated-headers)
+#include "gtest/gtest_prod.h"
```

Comment:
In Node-api meeting @gabrielschulhof  suggested this should be in "js_native_api_v8_internals.h" instead.